netlify/edge-functions/json.ts

```diff
@@ -1,5 +1,5 @@
 import type { Context } from "https://edge.netlify.com";
 
 export default async (request: Request, context: Context) => {
```

Comment:
Wondering whether we can remove the import of Context here given it's not used, or leave it in for consistency?

Reply:
I'd leave it in for consistency 🤷